layouts/section/job-posting.en.html

```diff
@@ -1,145 +1,154 @@
+  {{ if .Params.leverId }}
+  <div class="col-sm-10 col-sm-offset-1 col-xs-12">
+    <h2>Apply now</h2>
+    <form enctype="multipart/form-data" class="text-left" role="form" id="contactForm">
+      <input type="hidden" id="jobId" name="jobId" value="{{.Params.leverId}}" />
+      <!-- Full Name -->
+      <div class="form-group">
+        <label for="name">Full Name</label>
+        <input type="text" class="form-control validate-required" id="name" name="name" aria-required="true"
+          aria-describedby="name-required" />
+        <span id="name-required" class="error-message validate-required-message" style="display: none;">This field
+          is required</span>
+      </div>
+      <!-- Email -->
+      <div class="form-group">
+        <label for="email">Email</label>
+        <input type="email" name="email" class="form-control validate-required validate-email" id="email"
+          aria-required="true" aria-describedby="email-required" />
+        <span id="email-required" class="error-message validate-required-message" style="display: none;">This field
+          is required</span>
+      </div>
+
+      <!-- Resume -->
+
+      <label for="resume">Resume</label>
+      <div class="form-group">
+        <div class="browse">
+
+          <label>Browse...
+            <input class="resume" type="file" id="resume" name="resume"
+              accept="application/msword, application/pdf, text/plain" required style="display: none;" />
+          </label>
+
+        </div>
+        <span class="no-file-chosen" id="no-file-chosen-text">No file selected</span>
+
+      </div>
+
+      <!-- Body -->
+
+      <div class="form-group">
+        <label for="body">Tell us in 300 words or less why you want to work with us and,
+          if applicable, what we should know about your
+          work samples</label>
+        <textarea class="form-control validate-required validate-maxwords" id="body" name="body" rows="5"
+          aria-required="true" aria-describedby="body-required word-count-message"></textarea>
+        <span id="body-required" class="error-message validate-required-message" style="display: none;">This field
+          is required<br />
+        </span>
+        <span id="word-count-message" class="help-text">
+          <span id="word-count">300</span> words remaining</span>
+
+      </div>
+      <!-- Samples of your work -->
+      <div class="form-group">
+        <label for="samples">Samples of your work<span class="alt-label"> (optional)</span></label>
+        <div>
+          <span class="url">URLs, please!</span>
+
+        </div>
+
+        <textarea class="form-control validate-required validate-maxwords" id="samples" name="samples" rows="5"
+          aria-required="true" aria-describedby="body-required word-count-message"></textarea>
+      </div>
+      <!-- Submit-->
+      <div>
+        <button type="submit" class="submit-btn">
+          Submit<img src="/img/icons/arrow-right-white.svg" />
```

Comment:
If you want to not add alt text then you need to add either `role="none"` or `role="presentation"` to this element. 

I would go with presentation: 
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_presentation_role